xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/client/serverinfo/ReportingModeServerInfo.java

```diff
@@ -0,0 +1,298 @@
+
+  /**
+   * Current multiplier to the transaction cost based on load to this server.
+   *
+   * @return An optionally-present {@link BigDecimal} representing the local load factor.
+   */
+  @JsonProperty("load_factor_local")
+  Optional<BigDecimal> loadFactorLocal();
+
+  /**
+   * Current multiplier to the transaction cost being used by the rest of the network (estimated from other servers'
+   * reported load values).
+   *
+   * @return An optionally-present {@link BigDecimal} representing the network load factor.
+   */
+  @JsonProperty("load_factor_net")
+  Optional<BigDecimal> loadFactorNet();
+
+  /**
+   * Current multiplier to the transaction cost based on load to servers in this cluster.
+   *
+   * @return An optionally-present {@link BigDecimal} representing the cluster load factor.
+   */
+  @JsonProperty("load_factor_cluster")
+  Optional<BigDecimal> loadFactorCluster();
+
+  /**
+   * The current multiplier to the transaction cost that a transaction must pay to get into the open ledger.
+   *
+   * @return An optionally-present {@link BigDecimal} representing the open ledger load factor.
+   */
+  @JsonProperty("load_factor_fee_escalation")
+  Optional<BigDecimal> loadFactorFeeEscalation();
+
+  /**
+   * The current multiplier to the transaction cost that a transaction must pay to get into the queue, if the queue is
+   * full.
+   *
+   * @return An optionally-present {@link BigDecimal} representing the queue load factor.
+   */
+  @JsonProperty("load_factor_fee_queue")
+  Optional<BigDecimal> loadFactorFeeQueue();
+
+  /**
+   * The load factor the server is enforcing, not including the open ledger cost.
+   *
+   * @return An optionally-present {@link BigDecimal} representing the server load factor.
+   */
+  @JsonProperty("load_factor_server")
+  Optional<BigDecimal> loadFactorServer();
+
+  /**
+   * How many other rippled servers this one is currently connected to.
+   *
+   * @return An optionally-present {@link UnsignedInteger} representing the number of peers of this server.
+   */
+  Optional<UnsignedInteger> peers();
+
+  /**
+   * Public key used to verify this server for peer-to-peer communications. This node key pair is automatically
+   * generated by the server the first time it starts up. (If deleted, the server can create a new pair of keys.) You
+   * can set a persistent value in the config file using the {@code [node_seed]} config option, which is useful for
+   * clustering.
+   *
+   * @return A {@link String} containing the node's public key.
+   */
+  @JsonProperty("pubkey_node")
+  String publicKeyNode();
+
+  /**
+   * (Admin only) Public key used by this node to sign ledger validations. This validation key pair is derived from the
+   * {@code [validator_token]} or {@code [validation_seed]} config field.
+   *
+   * @return A {@link String} containing the validator's public key.
+   */
+  @JsonProperty("pubkey_validator")
+  Optional<String> publicKeyValidator();
```

Comment:
How do we know which properties in this class are optional vs not?

Reply:
We do not. I wanted to ask, should I just include only the ones we know are in there. I removed the ones which are not there for sure. Because, there is no documentation. We can keep it tight, I don't mind